server/api/file.go

```diff
@@ -28,7 +28,19 @@ import (
 	"github.com/woodpecker-ci/woodpecker/server/store"
 )
 
-// FileList gets a list file by pipeline.
+// FileList
```

Comment:
Those file endpoints are not used atm afaik and will be probably removed at some point.
Can we disable / skip those file endpoint from the swagger docs for now somehow.

Reply:
> Can we disable / skip those file endpoint from the swagger docs for now somehow.
Sure, just removing the whole API documentation section on the handler will make it disapear.
AND ... since you're phrasing "higher knowledge" I would recommend making a comment instead,
stating, why this should not be present in public documentation (and/or if there's a pre-condition like .e.g feature-x must be finished first.) 

The idea behind is to manage expectations of the API consumers. I mean, it's open source, people can look up endpoints (more or less) easily. So, better make expectations about API usage explicit.

Shall I remove all of these `/repos/{owner}/{name}/files*` endpoints from the API doc?
Could you suggest a reason OR pre-condition, by when it would make sense to document them?

Reply:
Its an unused system. I've just created a PR to remove it #1791

Reply:
@anbraten to avoid merge conflicts, I just reverted my changes in this file.go